embedded_sass.proto

```diff
@@ -117,6 +117,18 @@ message InboundMessage {
     // custom global functions. They must also reject any custom function names
     // that conflict with function names built into the Sass language.
     repeated string global_functions = 7;
+
+    // Possible ways to to format errors and logs.
+    message AlertStyle {
```

Comment:
I'd probably just inline these fields into `CompileRequest`, unless we anticipate re-using this elsewhere.